dev/gulpfile-wrap-template.js

```diff
@@ -0,0 +1,18 @@
+;(function(window, document, undefined) {
+  "use strict";
+  
+  <%= contents %>
```

Comment:
Expected an identifier and instead saw '<'.<br>Expected an operator and instead saw '%='.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected an identifier and instead saw '>'.<br>'contents' is not defined.